protocol/contracts/oracle/src/api/contract.rs

```diff
@@ -113,6 +115,17 @@ pub struct DispatchAlarmsResponse(pub AlarmsCount);
 
 pub type SupportedCurrencyPairsResponse = Vec<SwapLeg>;
 
+pub type CurrencyDefinitionsResponse = Vec<CurrencyDefinition>;
+
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+pub struct CurrencyDefinition {
+    pub ticker: String,
+    pub bank_symbol: String,
+    pub dex_symbol: String,
+    pub decimal_places: u8,
+}
```

Comment:
we also need to know the group it directly is a member of. They are Native, Lpns, Lease, and PaymentOnlyGroup